### ISSUE TITLE
Navball issues with flight plans

### ISSUE THREAD

**Issue by wingertge** [mockingbirdnest/Principia#1550]
Probably an incompatibility issue with one of my mods, so I uploaded a list of my installed mods here: https://pastebin.com/xz3DXHfN
I spent quite a while typing all 116 Mods with their version so you can CTRL+F them so hope it helps.

The issue always occurs when creating a flight plan and showing it on the NavBall. The burn time is calculated properly in the flight plan, but the NavBall shows either 0 or n/a. When I do actually burn, the dV value keeps resetting to the total (flickering between total and total-0.1) so I can't actually use that either.

There are no related errors in the logs, but here they are regardless.
https://pastebin.com/Jinu3yxA last ERROR log - warning and info are identical.

I'd look for the issue in the code myself, but my C++ is more than a little rusty so I doubt I'd get far.
Also a potentially related issue is the fact that the "Warp to ManNode" button shows up, but when warping to it Principia completely crashes, followed by a game freeze a few seconds later.
FATAL log here: https://pastebin.com/7RzSSZXq

System specs (shouldn't matter apart from OS, but it won't hurt I suppose)
Windows 10 (freshly installed yesterday)
Intel i7 4770K
nVidia GTX 1080 Ti
ASUS ROG Hero VI MB
32GB Kingston HyperX Low Profile RAM
Game runs off a 256GB SanDisk SSD
Sound Blaster ZxR Sound Card
Other specs don't interact with the game in any way.

If I forgot any info please do ask, I'll be happy to provide it.

**Comment by eggrobin** [comment]
Thanks for the report. There are two separate issues here.
The first part is that the stock Δv counter does not count down as you burn, which is known and documented, see [the relevant section of the concepts page](https://github.com/mockingbirdnest/Principia/wiki/Concepts#flight-planning-user-interface).

The second one is a crash that seems similar to #1549;  as mentioned [in the FAQ](https://github.com/mockingbirdnest/Principia/wiki/Installing,-reporting-bugs,-and-frequently-asked-questions#windows-dialog-box-or-sigabrt), we need the *full* INFO log, so please provide that. This is because it starts with additional information not contained in the ERROR log; without it the stack trace is useless.

**Comment by eggrobin** [comment]
A common element between this and #1549 seems to be the fact that you have KIS installed; can you provide details as to how you used KIS in the game where you encountered that crash?

**Comment by pleroy** [comment]
Looking into this further we are no longer sure that it is related to #1549.  We'll definitely need logs as explained by @eggrobin above.

**Comment by wingertge** [comment]
Yes, apologies. I thought I saw the file sizes being identical, but that was actually the file extension name, not the size. I'm an idiot. I also completely missed that part about the dV counter not going down - my bad on that as well.

Here's the INFO log for the crash period (this time on paste.ee, it's 32KB too large for pastebin): https://paste.ee/p/yPWkD

I'm also not sure if I should open a separate issue about the time to cutoff not being shown on the stock navball (as a feature request), since that would allow for automated burns by MechJeb and Co. I like to use those for things like RemoteTech satellite positioning and stuff, so it'd be nice to have support for them.

**Comment by pleroy** [comment]
For our reference, decoded stack trace:


[`ksp_plugin/vessel.cpp:115`](https://github.com/mockingbirdnest/Principia/blob/c9d0610117bbfaa4633fa3f2d5b72b99547f7151/ksp_plugin/vessel.cpp#L115)
[`ksp_plugin/plugin.cpp:484`](https://github.com/mockingbirdnest/Principia/blob/c9d0610117bbfaa4633fa3f2d5b72b99547f7151/ksp_plugin/plugin.cpp#L484)
[`ksp_plugin/interface.cpp:356`](https://github.com/mockingbirdnest/Principia/blob/c9d0610117bbfaa4633fa3f2d5b72b99547f7151/ksp_plugin/interface.cpp#L356)
